.github/workflows/maintenance_bot.yaml

```diff
@@ -10,6 +10,11 @@ jobs:
     env:
       MILESTONE_NUMBER: 19
     steps:
+      - name: Debug
```

Comment:
In case any one is wondering, I have been trying to understand why sometimes the bot removes manually-added labels, but I couldn't reproduce it reliably on my fork. So it's not a debugging leftover, I'd like to have these lines in for when it reappears.